tools/pipelines/templates/include-use-node-version.yml

```diff
@@ -7,7 +7,7 @@ parameters:
 # The node version required. The version must already be installed in the image.
 - name: version
   type: string
-  default: '18.17.1'
+  default: '18.20.7'
```

Comment:
Syncpack 13 [wants Node >=18.18.0](https://dev.azure.com/fluidframework/public/_build/results?buildId=325558&view=logs&j=0a267589-51b1-5666-1e58-38835ac8a8c1&t=1d230285-3682-58c8-415c-1f83253508a5&l=18). Might as well go to the latest. Not sure how long it'll take to complete the move to Node20 so this still seems worth doing. And I can update our build agent images to also have 18.20.7 baked in.

Reply:
Is this really how simple it is to use a new version? I asked repeatedly that it be made this simple and was told NOT POSSIBLE. It's really this easy?

Reply:
AFAIK yes, for the purpose of the pipelines using it when they run. I can't remember who added the option, I would have guessed you 😆 but maybe Andrei. Locally I think we'd want to update the engines in package.json, like I noticed you did in https://github.com/microsoft/FluidFramework/pull/22146 and I didn't here.

Reply:
It's also what @CraigMacomber did in https://github.com/microsoft/FluidFramework/pull/21965 to see how painful the move to Node20 would be.

Reply:
<sigh> I'm just frustrated because I did this upgrade months ago but couldn't proceed because CI was running an old node version, despite me making this same change in my PR. I even split the node upgrade out to [a separate change](https://github.com/microsoft/FluidFramework/pull/22371), but I was [so annoyed by the discussion](https://teams.microsoft.com/l/message/19:50292e8934024fc19d6ca2080dd7681e@thread.skype/1724776908757?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=9ce27575-2f82-4689-abdb-bcff07e8063b&parentMessageId=1724776908757&teamName=Fluid%20Framework&channelName=Dev&createdTime=1724776908757) that I eventually abandoned the PR. It's not clear to me why this didn't work 6 months ago, I guess.